frontend/src/components/common/info/messages/EmptyArchivedClassroomsMessage.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { type ReactElement } from "react";
+import { Center, useStyleConfig } from "@chakra-ui/react";
+
+import DisplayAssessmentsIllustration from "../../../../assets/illustrations/display-assessments.svg";
+import MessageContainer from "../MessageContainer";
+
+const EmptyArchivedClassroomsMessage = (): ReactElement => {
+  const styles = useStyleConfig("Center", { variant: "emptyMessage" });
+  return (
+    <Center __css={styles} height="100%" mt={4} pb={14}>
+      <MessageContainer
+        image={DisplayAssessmentsIllustration}
+        paragraphs={["Use the archive option on a classroom to view it here"]}
```

Comment:
i feel like this is unnecessary imo?

Reply:
I think it's better to be extra explicit here!